packages/pangea-sdk/pangea/services/authn/models.py

```diff
@@ -434,281 +287,205 @@ class ClientJWKSResult(PangeaResponseResult):
+    SMS_OTP = "sms_otp"
+    SOCIAL = "social"
+    TOTP = "totp"
+    VERIFY_EMAIL = "verify_email"
 
+    def __str__(self):
+        return self.value
 
-class FlowEnrollMFAStartResult(CommonFlowResult):
-    pass
+    def __repr__(self):
+        return self.value
 
 
-# - path: authn::/v1/flow/signup/password
-# https://pangea.cloud/docs/api/authn#password-sign-up
-class FlowSignupPasswordRequest(APIRequestModel):
-    flow_id: str
-    password: str
-    first_name: str
-    last_name: str
+class FlowRestartDataSMSOTP(APIRequestModel):
+    phone: str
 
 
-class FlowSignupPasswordResult(CommonFlowResult):
-    pass
+FlowRestartData = Union[Dict, FlowRestartDataSMSOTP]
 
 
-# - path: authn::/v1/flow/signup/social
-# https://pangea.cloud/docs/api/authn#social-sign-up
-class FlowSignupSocialRequest(APIRequestModel):
+class FlowRestartRequest(APIRequestModel):
     flow_id: str
-    cb_state: str
-    cb_code: str
+    choice: FlowChoice
+    data: FlowRestartData
 
 
-class FlowSignupSocialResult(CommonFlowResult):
+class FlowRestartResult(CommonFlowResult):
     pass
 
 
-# - path: authn::/v1/flow/start
-# https://pangea.cloud/docs/api/authn#start-a-sign-up-in
 class FlowStartRequest(APIRequestModel):
     cb_uri: Optional[str] = None
     email: Optional[str] = None
     flow_types: Optional[List[FlowType]] = None
-    provider: Optional[IDProvider] = None
     invitation: Optional[str] = None
 
 
 class FlowStartResult(CommonFlowResult):
     pass
 
 
-# - path: authn::/v1/flow/verify/captcha
-# https://pangea.cloud/docs/api/authn#verify-captcha
-class FlowVerifyCaptchaRequest(APIRequestModel):
-    flow_id: str
-    code: str
+class FlowUpdateDataAgreements(APIRequestModel):
+    agreed: List[str]
 
 
-class FlowVerifyCaptchaResult(CommonFlowResult):
-    pass
-
-
-# - path: authn::/v1/flow/verify/email
-# https://pangea.cloud/docs/api/authn#verify-email-address
-class FlowVerifyEmailRequest(APIRequestModel):
-    flow_id: str
-    cb_state: Optional[str] = None
-    cb_code: Optional[str] = None
-
-
-class FlowVerifyEmailResult(CommonFlowResult):
-    pass
-
-
-# - path: authn::/v1/flow/verify/mfa/complete
-# https://pangea.cloud/docs/api/authn#complete-mfa-verification
-class FlowVerifyMFACompleteRequest(APIRequestModel):
-    flow_id: str
-    code: Optional[str] = None
-    cancel: Optional[bool] = None
+class FlowUpdateDataCaptcha(APIRequestModel):
+    code: str
 
 
-class FlowVerifyMFACompleteResult(CommonFlowResult):
-    pass
+class FlowUpdateDataEmailOTP(APIResponseModel):
+    code: str
 
 
-# - path: authn::/v1/flow/verify/mfa/start
-# https://pangea.cloud/docs/api/authn#start-mfa-verification
-class FlowVerifyMFAStartRequest(APIRequestModel):
-    flow_id: str
-    mfa_provider: MFAProvider
+class FlowUpdateDataMagiclink(APIRequestModel):
+    state: str
+    code: str
 
 
-class FlowVerifyMFAStartResult(CommonFlowResult):
+class FlowUpdateDataPasskey(APIRequestModel):
+    # FIXME: is empty?
```

Comment:
passkey support is not official yet.
The passkey "update" data format contains either an "authentication" field, or a "registration" field.
We didnt nail down the format that these fields have yet, but here are some examples:
```
    "authentication": {
      "id": "URnBeUnim1qaQwLm-IiVhAL2Pi_OiHrn3A39K8AZSo0",
      "rawId": "URnBeUnim1qaQwLm-IiVhAL2Pi_OiHrn3A39K8AZSo0",
      "response": {
        "authenticatorData": "SZYN5YgOjGh0NBcPZHZgW4_krrmihjLHmVzzuoMdl2MFAAAAAA",
        "clientDataJSON": "eyJ0eXBlIjoid2ViYXV0aG4uZ2V0IiwiY2hhbGxlbmdlIjoiOGg1T3VrWlBLNjltSG5sZmYwOWZaTllzYXg3S1lSOEx4ZlItZUV4ZEZaWSIsIm9yaWdpbiI6Imh0dHA6Ly9sb2NhbGhvc3Q6MzAwMCIsImNyb3NzT3JpZ2luIjpmYWxzZX0",
        "signature": "MEQCIDAYFRrSceeKRaPfQBjMCgGLFY1wHspoBF4Y2eYdz8TTAiBUlM-43-I4H-y2qDG74y1-AtuJ3KukHnxF1g4Al2Cciw",
        "userHandle": "cHVpX3B5cTM3d3Rtcm1vbjdsbXJvdWt3cmlyMmtlemVhc3Jj"
      },
      "type": "public-key",
      "clientExtensionResults": {},
      "authenticatorAttachment": "platform"
    }
```

and
```
    "registration": {
      "id": "t7I3ITDlPiHdsvJs02-xYtQMCnvqarMQzsh4TlTU22A",
      "rawId": "t7I3ITDlPiHdsvJs02-xYtQMCnvqarMQzsh4TlTU22A",
      "response": {
        "attestationObject": "o2NmbXRkbm9uZWdhdHRTdG10oGhhdXRoRGF0YVikSZYN5YgOjGh0NBcPZHZgW4_krrmihjLHmVzzuoMdl2NFAAAAAK3OAAI1vMYKZIsLJfHwVQMAILeyNyEw5T4h3bLybNNvsWLUDAp76mqzEM7IeE5U1NtgpQECAyYgASFYIAwHubbbTzKj0S6x8Ohjq3jW0ipGJndtFb1nIJlMVUEBIlggxwHCTeG5AD9FY47044ty0RwzvZrPOUl9AXfxGjlSdI8",
        "clientDataJSON": "eyJ0eXBlIjoid2ViYXV0aG4uY3JlYXRlIiwiY2hhbGxlbmdlIjoiMVNMOVl3UDJmWmFzaWVFcTNxLWVBVGFNOXo2dE1LTUlvQld3NjZOY3ZSMCIsIm9yaWdpbiI6Imh0dHA6Ly9sb2NhbGhvc3Q6MzAwMCIsImNyb3NzT3JpZ2luIjpmYWxzZX0",
        "transports": [
          "internal"
        ],
        "publicKeyAlgorithm": -7,
        "publicKey": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEDAe5tttPMqPRLrHw6GOreNbSKkYmd20VvWcgmUxVQQHHAcJN4bkAP0VjjvTji3LRHDO9ms85SX0Bd_EaOVJ0jw",
        "authenticatorData": "SZYN5YgOjGh0NBcPZHZgW4_krrmihjLHmVzzuoMdl2NFAAAAAK3OAAI1vMYKZIsLJfHwVQMAILeyNyEw5T4h3bLybNNvsWLUDAp76mqzEM7IeE5U1NtgpQECAyYgASFYIAwHubbbTzKj0S6x8Ohjq3jW0ipGJndtFb1nIJlMVUEBIlggxwHCTeG5AD9FY47044ty0RwzvZrPOUl9AXfxGjlSdI8"
      },
      "type": "public-key",
      "clientExtensionResults": {},
      "authenticatorAttachment": "platform"
    }
```

Reply:
Will remove it then